Sources/Core/API/Scene.swift

```diff
@@ -229,7 +229,9 @@ open class Scene: Pluggable, Activatable {
         let wasWithinScene = actor.isWithinScene
         let sceneRect = Rect(origin: .zero, size: size)
         actor.isWithinScene = actor.rect.intersects(sceneRect)
-
+        if !wasWithinScene && actor.isWithinScene {
```

Comment:
Can you surround this if statement with new lines to make the code a bit easier to read? 🙂

Reply:
You are a nice helpful person John.  By new lines do you just mean blank lines?  I will implement this using else if as you suggest.  I have no idea of how to implement a test in ActorTests so I will work on that now.  I used your movement tutorial and removed the boundary constraints and added an observer to the players new entered scene event during movement phase and logged it into the debug area.

Reply:
Looks great! Thanks a lot for this contribution! 🎉 And looks like this is your first time doing open source so hope you'll keep at it 😀